recipes/pcaone/build.sh

```diff
@@ -2,26 +2,12 @@
 
 mkdir -p ${PREFIX}/bin
 # fix zlib issue
-export CFLAGS="${CFLAGS} -O3"
-export CPPFLAGS="${CPPFLAGS} -O3 -I${PREFIX}/include"
+export CFLAGS="${CFLAGS} -I${PREFIX}/include"
 export CXXFLAGS="${CXXFLAGS} -O3"
 export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
 
-sed -i.bak 's|-lpthread|-pthread|' Makefile
-rm -rf *.bak
+# sed -i.bak 's|-lpthread|-pthread|' Makefile
+# rm -rf *.bak
```

Comment:
If this isn't needed any more, just delete the lines.

Reply:
Thanks! just pushed a fix